models/ldndc/inst/template.job

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# redirect output
+exec 3>&1
+exec &> "@OUTDIR@/logfile.txt"
+
+# host specific setup
+@HOST_SETUP@
+
+# create output folder
+mkdir -p "@OUTDIR@"
+
+cd "@RUNDIR@"
+ln -s "@METPATH@" climate.txt
+
+
+# see if application needs running
+if [ ! -e "@OUTDIR@/results.csv" ]; then
```

Comment:
I believe LDNDC never produces `results.csv`, so every `runModule.start.model.runs` will re-run the successfully finished runs as well. Check if `Output` folder for the raw model outputs exist in your case 
```suggestion
if [ ! -e "@OUTDIR@/Output" ]; then
```